tools/Google.Cloud.Tools.ReleaseManager/ContainerCommands/IntegrationTestLibraryCommand.cs

```diff
@@ -44,4 +46,36 @@ public int Execute(ContainerOptions options)
         }
         return 0;
     }
+
+    private void MaybeSaveServiceAccount()
```

Comment:
This is fine, but I wonder if it ain't better to configure Docker to use the host's ADC setup when running the container locally?

Or maybe I'm misunderstanding something.

Reply:
That would be possible, but given the many ways that ADC can work, it could be pretty involved. If I've got an environment variable of GOOGLE_APPLICATION_CREDENTIALS defined to point to a file in some random directory, would we mount that directory, or copy the file, or something else? What about if it's just via gcloud? etc etc..

I'd be more inclined to put significant work into this if we expected running like this to be common - but currently I'm only expecting this to be used when developing either Librarian or a container.